### DISCUSSION TITLE
0071 XLS-71d: Initial Owner Reserve Exemption

### DISCUSSION THREAD

**Discussion by xVet** [XRPLF/XRPL-Standards#204] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by mvadari** [comment]
A couple of questions.
1. Your first two trustlines are free right now (a little-known feature). This is basically just an extension of that philosophy to all objects?
2. If I own 3 objects, do I owe 3 reserves or just 1? i.e. is the reserve exemption temporary?
3. Why roll it out slowly instead of in one amendment all at once?

**Reply by brettmollin** [reply]
re: 1, I think of it as "the first 2 trustlines can be unfunded" rather than "free". If you have enough XRP it is locked in reserve, if you don't have it you can still create 2 unfunded trustlines. Actually making it free could be less confusing.

**Reply by xVet** [reply]
> A couple of questions.
> 
> 1. Your first two trustlines are free right now (a little-known feature). This is basically just an extension of that philosophy to all objects?
> 
> 2. If I own 3 objects, do I owe 3 reserves or just 1? i.e. is the reserve exemption temporary?
> 
> 3. Why roll it out slowly instead of in one amendment all at once?

1. Correct

2. The reserve exemption is permanently from what i saw, though that's a good question to double check.

3. I would roll it out to as many as possible in one go but i want to leave room for people to defend the enforcing of owner reserve for specific object creating transactors if they deem it necessary

**Reply by mvadari** [reply]
2. It's not permanent IIRC, I believe you owe 3 reserves after you have 3 trustlines.

**Comment by mDuo13** [comment]
I'm supportive of this. The "two free trust lines" rule has been an obscure and little-used feature, but expanding it to all ledger types is good generalization, and this is a good opportunity to revisit the original design and make it less surprising.

I would personally not roll it out in multiple stages, but would go straight to, "You can own any two ledger objects with no reserve requirement" for simplicity.

**Reply by xVet** [reply]
> I'm supportive of this. The "two free trust lines" rule has been an obscure and little-used feature, but expanding it to all ledger types is good generalization, and this is a good opportunity to revisit the original design and make it less surprising.
> 
> 
> 
> I would personally not roll it out in multiple stages, but would go straight to, "You can own any two ledger objects with no reserve requirement" for simplicity.

Thank you @mDuo13 ! Good approach with skipping the stage rollout

**Comment by MarkusTeufelberger** [comment]
Maybe generalize it to "the first 2 owner reserves are free"? There might be features that cost more than 1 owner reserve per object in the future for some reason (e.g. very heavy cost computation or storage wise) and making any first 2 objects free might cause a price difference.

[a comment by an automated account is not shown]

**Reply by admin** [reply]
Already merged: https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0071-initial-owner-reserve-exemption
